docs/Using-Operations-Service.md

```diff
@@ -110,7 +110,7 @@ operationsService.listener = object: IOperationsServiceListener {
         // show UI the last fetch has failed
     }
 
-    override fun operationsLoaded(operations: List<UserOperation>) {
+    fun operationsChanged(operations: List<UserOperation>, removed: List<UserOperation>, added: List<UserOperation>) {
```

Comment:
Missing `override` keyword. It would be also good to note that the changes are based on the last call (you wrote it for parameters in the inline documentation).